src/sentry/static/sentry/app/components/actions/resolve.tsx

```diff
@@ -36,6 +37,7 @@ type Props = {
   disabled?: boolean;
   disableDropdown?: boolean;
   projectFetchError?: boolean;
+  inboxRowAction?: boolean;
```

Comment:
what about something slightly more descriptive like `showInboxAction` or `hasInboxAction`? Or maybe just `hasInbox`?

Reply:
`hasInbox` is probably the best